packages/type-orm/src/services/sorting.service.ts

```diff
@@ -31,7 +37,16 @@ export class SortingService {
 
     return sortingFields.reduce((acc: TypeORMOrdering, sortingField) => {
       const dbName = dbColumns[sortingField.name]
-      acc[`${prefix}${dbName}`] = sortingField.direction
+      acc[`${prefix}${dbName}`] = (sortingField.nulls) 
+        // complex ordering form that supports nulls direction
+        ? {
+          order: sortingField.direction, 
+          nulls: (sortingField.nulls === "FIRST") 
+            ? `NULLS FIRST` 
+            : `NULLS LAST`
```

Comment:
hmmmm...
I don't like this ternary inside another ternary... 
Why not just always return an `{ order, nulls? }` object ?

Reply:
done